### PR TITLE
SOC-275 | Dont load message if mw is empty

### PR DESCRIPTION
**Pull request by bkoval**
https://wikia-inc.atlassian.net/browse/SOC-275
@garthwebb @kvas-damian

### CODE REVIEW

**Comment by kvas-damian** [comment]
LGTM

File: extensions/wikia/Wall/WallExternalController.class.php [hunk 1]
```diff
@@ -675,24 +675,28 @@ public function getFormattedQuoteText() {
...
 			$username = $mw->getUser()->getName();
 
-			$convertToFormat = $this->request->getVal('convertToFormat', '');
+			$convertToFormat = $this->request->getVal( 'convertToFormat', '' );
 
-			if($convertToFormat == 'wikitext') {
-				$markup = '<div class="quote">' . "\n" . wfMsgForContent('wall-quote-author', $username) . "\n" . $mw->getRawText() . "\n</div>\n";
+			if ( $convertToFormat == 'wikitext' ) {
+				$markup = '<div class="quote">'
+					. "\n" . wfMsgForContent( 'wall-quote-author', $username )
```
Comment (garthwebb) [inline extensions/wikia/Wall/WallExternalController.class.php]: While your tidying up in here lets replace these deprecated wfMsg calls.  I think you could use:
```php
wfMessage( 'key', $args )->inContentLanguage()->escaped();
```

File: extensions/wikia/Wall/WallExternalController.class.php [hunk 2]
```diff
@@ -675,24 +675,28 @@ public function getFormattedQuoteText() {
...
+					. "\n" . $mw->getRawText() . "\n</div>\n";
 			} else {
-				$markup = $this->getConvertedContent('<div class="quote">' . wfMsgForContent('wall-quote-author', $username) . "<br>" . $mw->getRawText() . "\n</div><br>");
+				$markup = $this->getConvertedContent( '<div class="quote">'
+					. wfMsgForContent( 'wall-quote-author', $username )
```
Comment (garthwebb) [inline extensions/wikia/Wall/WallExternalController.class.php]: Same as above

**Comment by armonr** [comment]
:+1: 
Besides comments from Garth, :shipit:

**Pull request merged**